database_schemas/DB_generation_schema.sql

```diff
@@ -1,131 +1,176 @@
+  (
+     dataconf_id    CHAR(36) NOT NULL,
+     quantile_value NUMERIC(3, 2) NOT NULL
+  );
+
+ALTER TABLE dataset_quantile
+  ADD CONSTRAINT dataset_quantile_pk PRIMARY KEY (dataconf_id);
 
 -- Add foreign key constraints to tables
-
 ALTER TABLE upload
-    ADD CONSTRAINT rabit_user_upload
-        FOREIGN KEY (user_id) REFERENCES rabit_user (user_id)
-            ON DELETE CASCADE;
+  ADD CONSTRAINT rabit_user_upload FOREIGN KEY (user_id) REFERENCES rabit_user (
+  user_id) ON DELETE CASCADE;
 
 ALTER TABLE file_pointer
-    ADD CONSTRAINT upload_file_pointer
-        FOREIGN KEY (upload_id) REFERENCES upload (upload_id)
-            ON DELETE CASCADE;
+  ADD CONSTRAINT upload_file_pointer FOREIGN KEY (upload_id) REFERENCES upload (
+  upload_id) ON DELETE CASCADE;
 
 ALTER TABLE file_pointer
-    ADD CONSTRAINT plot_collection_file_pointer
-        FOREIGN KEY (collection_id) REFERENCES plot_collection (collection_id)
-            ON DELETE CASCADE;
+  ADD CONSTRAINT plot_collection_file_pointer FOREIGN KEY (collection_id)
+  REFERENCES plot_collection (collection_id) ON DELETE CASCADE;
 
 ALTER TABLE base_parameter
-    ADD CONSTRAINT plot_collection_base_parameter
-        FOREIGN KEY (collection_id) REFERENCES plot_collection (collection_id)
-            ON DELETE CASCADE;
+  ADD CONSTRAINT file_id_base_parameter FOREIGN KEY (file_id) REFERENCES
+  file_pointer (file_id) ON DELETE CASCADE;
 
 ALTER TABLE corner_plot
-    ADD CONSTRAINT rabit_user_corner_plot
-        FOREIGN KEY (user_id) REFERENCES rabit_user (user_id)
-            ON DELETE CASCADE;
+  ADD CONSTRAINT rabit_user_corner_plot FOREIGN KEY (user_id) REFERENCES
+  rabit_user (user_id) ON DELETE CASCADE;
 
 ALTER TABLE corner_plot
-    ADD CONSTRAINT plot_collection_corner_plot
-        FOREIGN KEY (collection_id) REFERENCES plot_collection (collection_id)
-            ON DELETE CASCADE;
+  ADD CONSTRAINT plot_collection_corner_plot FOREIGN KEY (collection_id)
+  REFERENCES plot_collection (collection_id) ON DELETE CASCADE;
 
 ALTER TABLE parameter_config
-    ADD CONSTRAINT corner_plot_parameter_config
-        FOREIGN KEY (corner_id) REFERENCES corner_plot (corner_id)
-            ON DELETE CASCADE;
+  ADD CONSTRAINT corner_plot_parameter_config FOREIGN KEY (corner_id) REFERENCES
+  corner_plot (corner_id) ON DELETE CASCADE;
 
 ALTER TABLE parameter_config
-    ADD CONSTRAINT base_parameter_parameter_config
-        FOREIGN KEY (parameter_id) REFERENCES base_parameter (parameter_id)
-            ON DELETE CASCADE;
+  ADD CONSTRAINT base_parameter_parameter_config FOREIGN KEY (parameter_id)
+  REFERENCES base_parameter (parameter_id) ON DELETE CASCADE;
+
+ALTER TABLE dataset_config
+  ADD CONSTRAINT dataset_config_file_pointer FOREIGN KEY (file_id) REFERENCES
+  file_pointer (file_id) ON DELETE CASCADE;
+
+ALTER TABLE dataset_config
+  ADD CONSTRAINT dataset_config_corner_plot FOREIGN KEY (corner_id) REFERENCES
+  corner_plot (corner_id) ON DELETE CASCADE;  
```

Comment:
```suggestion
  corner_plot (corner_id) ON DELETE CASCADE;  

ALTER TABLE dataset_sigma
  ADD CONSTRAINT dataset_sigma_dataset_config FOREIGN KEY (dataconf_id) REFERENCES
  dataset_config (corner_id) ON DELETE CASCADE;

ALTER TABLE dataset_quantile
  ADD CONSTRAINT dataset_quantile_dataset_config FOREIGN KEY (dataconf_id) REFERENCES
  dataset_config (corner_id) ON DELETE CASCADE;

```
Missing foreign key constraints for tables `dataset_sigma` and `dataset_quantile`